### ISSUE TITLE
Upgrade to Guzzle 4

### ISSUE THREAD

**Issue by cpliakas** [cpliakas/magento-client-php#2]

**Comment by stovak** [comment]
How quickly do you see yourself doing this? Was pondering using it for integration into our marketing site.

**Comment by stovak** [comment]
Just noticed the year. I'm going to fork this and attempt to update.

**Comment by cpliakas** [comment]
Approaching the 1 year anniversary. As solutions evolved, my need for this has actually gone away so I am not actively hacking at it. Happy to give you commit access to unblock you, I want it to be useful and don't want to be a barrier.

**Comment by stovak** [comment]
I'll play with the fork and get back to you

**Comment by cpliakas** [comment]
Great, also please let me know if it is crap and no longer useful :-). Definitely open to that feedback as well.

**Comment by bslinger** [comment]
@stovak Did you get anywhere with upgrading this to a more recent version of Guzzle?

**Comment by stovak** [comment]
Yes. Its checked into my local repo. I'm currently experimenting with using guzzle-services to define the API with nested arrays then instantiate the description and start making calls. Like 300% easier and only takes a few lines of code.

**Comment by cpliakas** [comment]
guzzle-services is awesome, great direction.
